src/file_utils.cr

```diff
@@ -38,39 +38,10 @@ module FileUtils
   # File.write("bar.cr", "1")
   # FileUtils.cmp("file.cr", "bar.cr") # => true
   # ```
-  def cmp(filename1 : String, filename2 : String)
-    return false unless File.size(filename1) == File.size(filename2)
-
-    File.open(filename1, "rb") do |file1|
-      File.open(filename2, "rb") do |file2|
-        cmp(file1, file2)
-      end
-    end
-  end
-
-  # Compares two streams *stream1* to *stream2* to determine if they are identical.
-  # Returns `true` if content are the same, `false` otherwise.
   #
-  # ```
-  # require "file_utils"
-  #
-  # File.write("afile", "123")
-  # stream1 = File.open("afile")
-  # stream2 = IO::Memory.new("123")
-  # FileUtils.cmp(stream1, stream2) # => true
-  # ```
-  def cmp(stream1 : IO, stream2 : IO)
-    buf1 = uninitialized UInt8[1024]
-    buf2 = uninitialized UInt8[1024]
-
-    while true
-      read1 = stream1.read(buf1.to_slice)
-      read2 = stream2.read_fully?(buf2.to_slice[0, read1])
-      return false unless read2
-
-      return false if buf1.to_unsafe.memcmp(buf2.to_unsafe, read1) != 0
-      return true if read1 == 0
-    end
+  # NOTE: Alias of `File.same_content?`
+  def cmp(filename1 : String, filename2 : String)
```

Comment:
I didn't know `cmp` was a command. Same here, I also think it can be deprecated.

Reply:
I get that you want to deprecate the whole FileUtils module. But why you said this method can be deprecated specifically?

Reply:
`cmp` is not very common, and quite cryptic name. Not sure this alias really helps.